src/Input.fs

```diff
@@ -45,7 +45,11 @@ module Input =
         | Valid of bool
         | Invalid of bool
         | Plaintext of string
+        | Placeholder of string
```

Comment:
As a rule of thumb, I wish to put everything that counts as an existing IHTMLProp (from https://github.com/fable-compiler/fable-react/blob/master/src/Fable.React.Props.fs) under `| Custom of IHTMLProp list` ([ex](https://github.com/nojaf/fable-reactstrap/blob/master/src/Input.fs#L53)).

This is to avoid collision with IHTMLProp.